src/Sulu/Bundle/WebsiteBundle/Controller/ExceptionController.php

```diff
@@ -47,47 +73,35 @@ public function showAction(
         }
 
         if ($request->getRequestFormat() === 'html') {
-            if ($exception->getStatusCode() == 404) {
-                return new Response(
-                    $this->twig->render(
-                        'ClientWebsiteBundle:views:error404.html.twig',
-                        array(
-                            'request' => array(
-                                'webspaceKey' => $this->requestAnalyzer->getWebspace()->getKey(),
-                                'locale' => $this->requestAnalyzer->getCurrentLocalization(),
-                                'portalUrl' => $this->requestAnalyzer->getPortalUrl(),
-                                'resourceLocatorPrefix' => $this->requestAnalyzer->getResourceLocatorPrefix(),
-                                'resourceLocator' => $this->requestAnalyzer->getResourceLocator(),
-                                'get' => $this->requestAnalyzer->getGetParameters(),
-                                'post' => $this->requestAnalyzer->getPostParameters(),
-                                'analyticsKey' => $this->requestAnalyzer->getAnalyticsKey()
-                            ),
-                            'path' => $request->getPathInfo()
-                        )
-                    ),
-                    404
-                );
-            } elseif ($exception->getStatusCode() < 500) {
-                $currentContent = $this->getAndCleanOutputBuffering($request->headers->get('X-Php-Ob-Level', -1));
-                $code = $exception->getStatusCode();
+            $currentContent = $this->getAndCleanOutputBuffering($request->headers->get('X-Php-Ob-Level', -1));
+            $code = $exception->getStatusCode();
+
+            // symfony exception template uses status_code and status_text
+            $parameter = array(
+                'statusCode' => $code,
+                'status_code' => $code,
+                'statusText' => isset(Response::$statusTexts[$code]) ? Response::$statusTexts[$code] : '',
+                'status_text' => isset(Response::$statusTexts[$code]) ? Response::$statusTexts[$code] : '',
```

Comment:
My suggestion would have been to use the names from symfony... And maybe also use `current_content` then? /cc @chirimoya